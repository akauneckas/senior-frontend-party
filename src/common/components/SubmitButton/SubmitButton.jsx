import React from 'react'
import PropTypes from 'prop-types';
import './SubmitButton.scss';

const propTypes = {
    children: PropTypes.string.isRequired,
};

const SubmitButton = ({ children }) => (
    <button type="submit" className="SubmitButton btn">{ children }</button>
);

SubmitButton.propTypes = propTypes;

export default SubmitButton;
