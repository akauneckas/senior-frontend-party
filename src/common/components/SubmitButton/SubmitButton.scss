.SubmitButton {
  &.btn {
    line-height: 2.75;
    color: #fff;
    background-color: #9fd533;
    border-color: #9fd533;
  }
  &:hover {
    background-color: #86b308;
    border-color: #86b308;
  }
}