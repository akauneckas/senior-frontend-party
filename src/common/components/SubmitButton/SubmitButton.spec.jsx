import React from 'react';
import renderer from 'react-test-renderer';
import SubmitButton from './SubmitButton';

describe('<SubmitButton />', () => {
    it('should match snapshot', () => {
        const component = renderer.create(
            <SubmitButton>
                Log in
            </SubmitButton>
        );
        expect(component.toJSON()).toMatchSnapshot();
    });
});
