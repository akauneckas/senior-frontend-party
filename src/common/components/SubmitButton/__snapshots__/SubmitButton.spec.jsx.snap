// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SubmitButton /> should match snapshot 1`] = `
<button
  className="SubmitButton btn"
  type="submit"
>
  Log in
</button>
`;
